### PR TITLE
NSM-14360  Added command to build the cache after adding node through API

### PR DESCRIPTION
**Pull request by chuklov**
Update to the test. Added karaf command to sync interface to node cache. 

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-14360

### CODE REVIEW

**Review by christianpape — APPROVED** [review]
LGTM.

**Review by fooker — CHANGES_REQUESTED** [review]
@chuklov did you already create an issue to tackle the underlying problem? If yes, please add a comment referencing the issue.

**Comment by chuklov** [comment]
> @chuklov did you already create an issue to tackle the underlying problem? If yes, please add a comment referencing the issue.
Here is the bug ticket for this.
https://issues.opennms.org/browse/NMS-14720

File: smoke-test/src/test/java/org/opennms/smoketest/flow/BmpIT.java [hunk 1]
```diff
@@ -173,6 +174,9 @@ public void verifyBmpProcessing() {
         final EventBuilder builder = new EventBuilder(EventConstants.RELOAD_DAEMON_CONFIG_UEI, getClass().getSimpleName());
         builder.setParam(EventConstants.PARM_DAEMON_NAME, "collectd");
         stack.opennms().getRestClient().sendEvent(builder.getEvent());
+        // Added command to build the cache after adding node through API
+        KarafShell karafShell = new KarafShell(stack.opennms().getSshAddress());
+        karafShell.runCommand("opennms:sync-node-cache");
```
Comment (RangerRick) [inline smoke-test/src/test/java/org/opennms/smoketest/flow/BmpIT.java:179]: aha! that's a clever way to do this, nice

**Review by fooker — APPROVED** [review]

**Pull request merged**